tests/unit/App.spec.js

```diff
@@ -0,0 +1,24 @@
+import { mount } from "@vue/test-utils";
+import { createRouter, createMemoryHistory } from "vue-router";
+import { routes } from "@/router";
+import App from "@/App";
+
+describe("App.vue", () => {
+  it("should render with default values", async () => {
+    const router = createRouter({
+      history: createMemoryHistory(),
+      routes,
+    });
```

Comment:
We're creating a new router object here right?
This will include the routes but this will not include the router navigation guards right?